src/Microsoft.ML.Data/Evaluators/MulticlassClassificationEvaluator.cs

```diff
@@ -246,9 +262,11 @@ public sealed class Counters
                 private double _totalLogLoss;
                 private double _numInstances;
                 private double _numCorrect;
-                private double _numCorrectTopK;
+                private int _numUnknownClassInstances;
```

Comment:
`_numUnknownClassInstances` should be a double, so it can track weighted instances. 

We may also not need the variable -- https://github.com/dotnet/machinelearning/pull/5395/files#r538314447

Reply:
Removed the variable as per the comment above